frontend/utils/test/testModels.ts

```diff
@@ -21,11 +21,11 @@ const accessRequestSchemaId = 'my-request-schema'
 const modelcardSchemaId = 'my-model-schema'
 
 export const testAccessRequest: AccessRequestInterface = {
+  _id: '123125123',
```

Comment:
Not super important, but could we make this more readable by giving it a value like `testAccessRequestId`. This would make it easier to see how the test data is linked to one another.